cmd/storj-sim/network.go

```diff
@@ -284,6 +285,13 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			},
 			"run": {},
 		})
+		var schema string
+		if flags.Postgres != "" {
+			schema = fmt.Sprintf("satellite%d", i)
+			postgresConnstr := pgutil.ConnstrWithSchema(flags.Postgres, schema)
+			dbArgs := []string{"--database", postgresConnstr, "--metainfo.database-url", postgresConnstr}
```

Comment:
main database and metainfo probably should use different schemas.
e.g. `satellite/0` and `satellite/0/metainfo`

Reply:
good point

Reply:
They may have some conflicts due to separate initialization. Might be better to leave them separate.

Reply:
what part of the initialization will conflict?  the createSchema function executes `create schema if not exists` so that should be ok. what else would conflict?

Reply:
There are other tables that the postgreskv makes and satellite. I'm unsure whether they conflict... easier it's to use different schemas to ensure it definitely doesn't happen. However, if it definitely doesn't conflict, then I'm fine with keeping them in the same schema.